src/Test/Utilities/CLRHelpers.cs

```diff
@@ -64,12 +65,14 @@ public static string[] PeVerify(string filePath)
             return PeVerify(File.ReadAllBytes(filePath), AppDomain.CurrentDomain.Id, filePath);
         }
 
-        private static readonly object Guard = new object();
+        private static readonly Mutex s_guard = new Mutex(false, new Guid().ToString());
```

Comment:
A comment about why a Mutex is preferable to a lock object would be useful--it isn't at all clear to me why this change was made.